packages/build-utils/test/integration.test.ts

```diff
@@ -31,6 +31,7 @@ const skipFixtures: string[] = [
   '07-zero-config-jekyll',
   '08-zero-config-middleman',
   '21-npm-workspaces',
+  '23-pnpm-workspaces',
```

Comment:
If we're not deploying, then how do we know it works correctly?

Reply:
> If we're not deploying, then how do we know it works correctly?

Ah that must be my misunderstanding. Do these integration tests not execute pnpm install when they are deployed?

> The unit tests don't actually run pnpm install so I think we need a test to verify that it works properly with the given flags.

I'll see if I can write some unit tests that execute `pnpm install`

Reply:
Look at the `20-npm-7` test fixture for an example text that actually deploys.

Note the `probes` are the parts that actually test the deployment req/res.

https://github.com/vercel/vercel/blob/6ef2c16d631c28e6282df473faefa44165031ef5/packages/build-utils/test/fixtures/20-npm-7/vercel.json#L1-L5